pages/cloudflare/index.mdx

```diff
@@ -53,7 +53,7 @@ We will update the list as we progress towards releasing 1.0.
 - [x] [Image optimization](https://nextjs.org/docs/app/building-your-application/optimizing/images) (you can integrate Cloudflare Images with Next.js by following [this guide](https://developers.cloudflare.com/images/transform-images/integrate-with-frameworks/))
 - [ ] [Pages Router](https://nextjs.org/docs/pages)
 - [ ] [Incremental Static Regeneration (ISR)](https://nextjs.org/docs/app/building-your-application/data-fetching/incremental-static-regeneration)
-- [ ] [Partial Prerendering (PPR)](https://nextjs.org/docs/app/building-your-application/rendering/partial-prerendering)
+- [x] [Partial Prerendering (PPR)](https://nextjs.org/docs/app/building-your-application/rendering/partial-prerendering)
```

Comment:
all the checked items are at the top, and the unchecked ones at the bottom, I am not sure but I feel like that was done intentionally? maybe it'd be cleaner to keep the convention and move this up? (is it easier to read?)

not a huge deal anyways 👍